app/config/parameters.yml.dist

```diff
@@ -75,6 +75,7 @@ parameters:
     env(APP_EMAIL_SUBSCRIPTIONS_WEBHOOK_SECRET): 'mailchimp_secret'
 
     env(MAILCHIMP_API_KEY): ''
+    env(MAILCHIMP_WEBHOOK_KEY): 'abc'
```

Comment:
Not a big deal and not a request change, but WDYT to use `webhook_key` for the default dist parameter ?